packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
>This change doesn't make sense because it's already metrics.

It looks really similar to `List.toList()` and follow the Dart usage rule  [PREFER naming a method to___() if it copies the object’s state to a new object.](https://www.dartlang.org/guides/language/effective-dart/design#prefer-naming-a-method-to___-if-it-copies-the-objects-state-to-a-new-object).

It's indeed a breaking change but quite limited because I don't think this method is highly used.

If you really prefer to avoid this change I can ignore this method with an `//ignore: ...`. I can also completely disable this rule in the yaml file. Let me know what you prefer.